modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFDSDataProviderCheck.java

```diff
@@ -172,14 +184,57 @@ private String _reorderGetItemsCount(String methodCall) {
 				parameterList.get(0)));
 	}
 
+	private String _updateImplementedClass(String content) {
+		Matcher matcher = _implementedClassPattern.matcher(content);
+
+		if (matcher.find()) {
+			return StringUtil.replace(
+				content, matcher.group(), matcher.group(1) + "FDSDataProvider");
+		}
+
+		return content;
+	}
```

Comment:
Hi @kyle-miho, since the imports.txt will be executed before, this will never happen, will it?